pkg/services/serviceaccounts/api/api.go

```diff
@@ -130,7 +155,10 @@ func (api *ServiceAccountsAPI) RetrieveServiceAccount(ctx *contextmodel.ReqConte
 		return response.Error(http.StatusBadRequest, "Service Account ID is invalid", err)
 	}
 
-	serviceAccount, err := api.service.RetrieveServiceAccount(ctx.Req.Context(), ctx.SignedInUser.GetOrgID(), scopeID)
+	serviceAccount, err := api.service.RetrieveServiceAccount(ctx.Req.Context(), &serviceaccounts.GetServiceAccountQuery{
+		OrgID: ctx.SignedInUser.GetOrgID(),
+		ID:    scopeID,
```

Comment:
I know it was here before but scopeID is a weird name for this one, what about just id?